internal/apiserver/route_get_net_did_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/hyperledger/firefly/mocks/networkmapmocks"
+	"github.com/hyperledger/firefly/pkg/fftypes"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestGetIdentityByDID(t *testing.T) {
+	o, r := newTestAPIServer()
+	nmn := &networkmapmocks.Manager{}
+	o.On("NetworkMap").Return(nmn)
+	req := httptest.NewRequest("GET", "/api/v1/network/did?did=did:firefly:org/org_1", nil)
```

Comment:
See this looks weird 🙃

Reply:
Was reading https://w3c-ccg.github.io/did-resolution/#example-0

Wondering about:
`/api/v1/network/identities/did:firefly:org/org_1`

I think that could work. Given @awrichar is out on vacation, I might make this proposal in code for @nguyer to review.

Reply:
Ok - that's done @nguyer, including sorting the swagger generator so it can handle the custom Gorilla mux syntax.